### PR TITLE
Feature / Download component

### PR DESCRIPTION
**Pull request by enguerranws**

### CODE REVIEW

**Comment by garronej** [comment]
Super merci! @enguerranws

File: package.json [hunk 1]
```diff
@@ -148,6 +148,8 @@
         "./Highlight": "./dist/Highlight.js",
         "./Header": "./dist/Header.js",
         "./Footer": "./dist/Footer.js",
+        "./File": "./dist/File.js",
+        "./Download": "./dist/Download.js",
         "./Display": "./dist/Display.js",
```
Comment (garronej) [inline package.json:153]: It's automatically generated, you don't need to worry about that.

File: src/Download.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import React, { memo, forwardRef, type CSSProperties } from "react";
+import { symToStr } from "tsafe/symToStr";
+import { assert } from "tsafe/assert";
+import type { Equals } from "tsafe";
+import { fr } from "./fr";
+import { cx } from "./tools/cx";
+import { getLink } from "./link";
+import { RegisteredLinkProps } from "./link";
```
Comment (garronej) [inline src/Download.tsx:8]: Always import types as `type`

Comment (garronej) [inline src/Download.tsx:8]:
```suggestion
import { getLink, type RegisteredLinkProps } from "./link";
```

File: src/Download.tsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+export type DownloadProps = {
+    className?: string;
+    style?: CSSProperties;
+    details: string;
+    label: string;
```
Comment (garronej) [inline src/Download.tsx:14]: Thoses two can be `ReactNode`

File: src/Download.tsx (hunk 2 above)
Comment (enguerranws) [reply src/Download.tsx:8]: Sorry, just forgot to review my own PR

Comment (garronej) [reply src/Download.tsx:8]: No problem!

**Pull request merged**